### DISCUSSION TITLE
Any Github API to interact discussions from code ?

### DISCUSSION THREAD

**Discussion by haseeb-xd** [community/community#38135] [Discussions]
Do we have any kind of API from which we can get the discussions, their replies and all other things?

**Comment by david-engelmann** [comment]
[There are webhooks for interactions, as well as commits and other aspects of a repo](https://docs.github.com/en/rest/interactions/repos). That documentation is well organized and has sample request for all the endpoints. Best of Luck!

**Comment by airtower-luna** [comment]
There is, but only in the GraphQL API: https://docs.github.com/en/graphql/reference/objects#discussion

There's no REST API for discussions, which has annoyed me, too. :sweat_smile:
